src/index.ts

```diff
@@ -17,6 +31,11 @@ router.all("*", async () => {
 
 export default {
   async fetch(request: Request, env: Env): Promise<Response> {
+    const isVerifiedRequest = await verifyBot(request, env);
```

Comment:
NIT: `isRequestVerified` sounds better

Reply:
I am not sure about this as earlier I used the suggested name but that did not sound better.